.circleci/config.yml

```diff
@@ -1,52 +1,74 @@
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/ruby:2.5.1-node-browsers
```

Comment:
I don't think we need the ruby image with node browsers included since it is an api